app/controllers/scrapers_controller.rb

```diff
@@ -0,0 +1,29 @@
+class ScrapersController < ApplicationController
```

Comment:
This is really cool that you can configure scrapers like this!  Is this something that any user can do or only an admin?  If only admins, let's talk tomorrow about restricting certain users access.